packages/manager/modules/vps/src/veeam/vps-veeam.routing.js

```diff
@@ -9,13 +10,40 @@ export default /* @ngInject */ ($stateProvider) => {
       },
     },
     resolve: {
+      automatedBackupOption: /* @ngInject */ (
+        $transition$,
+        hasAutomatedBackupOption,
+        VpsService,
+      ) =>
+        hasAutomatedBackupOption
+          ? VpsService.getVeeamInfo($transition$.params().serviceName)
+          : null,
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('vps_veeam'),
       goToVeeamRestore: /* @ngInject */ ($state) => (restorePoint) =>
         $state.go('vps.detail.veeam.restore', { restorePoint }),
       goToVeeamMount: /* @ngInject */ ($state) => (restorePoint, mount) =>
         $state.go('vps.detail.veeam.mount', { restorePoint, mount }),
+      goToVeeamScheduleBackup: /* @ngInject */ ($state) => () =>
+        $state.go('vps.detail.veeam.schedule-backup'),
+      goToVpsUpgrade: /* @ngInject */ ($state) => () =>
+        $state.go('vps.detail.dashboard.configuration.upgrade'),
       goBack: /* @ngInject */ ($state) => () => $state.go('vps.detail.veeam'),
-      breadcrumb: /* @ngInject */ ($translate) =>
-        $translate.instant('vps_veeam'),
+      hasAutomatedBackupOption: /* @ngInject */ ($transition$, VpsService) =>
+        VpsService.getVpsOption(
```

Comment:
Shouldn't we check the `automateBackupAvailability` before making one more call to the API ?

Reply:
Right, i will update it

Reply:
Fixed.